Chapter 2 corrections carlos (#138)
chhoumann/p7: report/sections/Preliminaries/preliminaries.tex

Where effectiveness regards how well the users accomplished their goal, efficiency regards the resources expended in relation to the completeness of the goals, and lastly, satisfaction regards the comfort of the system to the users.
To test usability, one may perform a usability test. This is a study of the interaction that users have with a product. It is done to identify usability problems in a system, and results in a list of usability problems and knowledge about how easily a user is able to interact with the system.
Before the test, the interviewers have to ensure that a test plan is ready.

> They also need to prepare tasks and questions for the test subjects, and ensure that the equipment works.

Reviewer: What is the equipment? Maybe say necessary equipment is available and functioning? My next question is then what equipment is needed.

> They also need to prepare tasks and questions for the test subjects, and ensure that the equipment works.

Author: ```suggestion
They also need to prepare tasks and questions for the test subjects, and ensure that the necessary equipment works.
```

During the test, representative users should interact with the design while solving tasks and thinking out loud. The interviewer collects data during this process. It is important that the interviewer interferes as little as possible by only providing help if the user is stuck.

> \cite{deb7} We use this method along with the system usability scale to perform a usability test on our system.

Reviewer: ```suggestion
\cite{deb7} We use this method along with SUS  to perform a usability test on our system.
```

	\item For even items: summarize the total score, then subtract the total score from 25
	\item Add up the total scores and multiply that total by 2.5.
\end{enumerate}
If the system achieves a SUS score above 68, it means that the UI is above average. Otherwise, if it is below 68, it is below average\cite{adobeSUS}.
We use SUS in order to get an overall rating of our system.

> \section*{BenchmarkDotnet}

Reviewer: ```suggestion
\section{BenchmarkDotnet}
```
